lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/logging/SessionInterpolatedSimpleValues.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.base.logging;
+
+import io.trino.plugin.base.logging.FormatInterpolator.InterpolatedValue;
+import io.trino.spi.connector.ConnectorSession;
+
+import java.util.function.Function;
+
+public enum SessionInterpolatedSimpleValues
```

Comment:
This is copied and modified `SessionInterpolatedValues` since the original object is also used in BigQuery

Reply:
it's ok to limit the scope by not fixing for BigQuery but same problem still exists in connectors who use `SessionInterpolatedValues`.

Reply:
Will create a follow-up ticket then and put it in the backlog

Reply:
I think you can simply remove `io.trino.plugin.base.logging.SessionInterpolatedValues.SanitizedValuesProvider` from `SessionInterpolatedValues`. I think the case in BigQuery is safe to be changed. There is no way to inject SQL as it is used only to set the label.